### ISSUE TITLE
Lib dependency not found

### ISSUE THREAD

**Issue by alvarosaburido** [ZachJW34/nx-plus#53]
Hello, first of all awesome work with this package. 🎉

I'm having an issue importing a library that I have under libs, that works perfectly fine in other Angular clients inside `apps`  but Fails in a  Vue client.

```
This dependency was not found:

* @my-awesome-monorepo/api-services in ./apps/vue-app/src/store/index.ts

To install it, you can run: npm install --save @my-awesome-monorepo/api-services
Build failed with errors.
```

I have it configure the path inside of the `tsconfig.json` 

```
{
  "compileOnSave": false,
  "compilerOptions": {
    ...
   "paths": {
      "@my-awesome-monorepo/api-services": ["libs/api-services/src/index.ts"],
   }
  }
}
```

It works if I use a relative path, but in this case inside of `libs/api-services/src/index.ts`  I have also imports with `@my-awesome-monorepo` paths for other libraries.

**Comment by ZachJW34** [comment]
Thanks @alvarosaburido!! Glad you're liking it.
Can you provide the steps to reproduce this error and the versions of the dependencies used? A minimal Github repo would be a great help as well!

**Comment by alvarosaburido** [comment]
Sure thing.

## Steps
- Clone [Repo](https://github.com/alvarosaburido/nx-plus-monorepo) `git-clone git@github.com:alvarosaburido/nx-plus-monorepo.git` 
- Run `nx serve vue-lab` 

## Expected result

`import awesome from '@my-awesome-monorepo/api'` inside `apps/vue-lab/src/App.vue` should work.

## Actual result

```

This dependency was not found:

* @my-awesome-monorepo/api in ./node_modules/cache-loader/dist/cjs.js??ref--12-0!./node_modules/@vue/cli-plugin-typescript/node_modules/ts-loader??ref--12-1!./node_modules/cache-loader/dist/cjs.js??ref--0-0!./node_modules/vue-loader/lib??vue-loader-options!./apps/vue-lab/src/App.vue?vue&type=script&lang=ts&

To install it, you can run: npm install --save @my-awesome-monorepo/api
```

**Comment by ZachJW34** [comment]
Thanks for providing the repo. The issue you are seeing is due to having a `vue.config.js` inside your application which overrides the changes we make in our vue builders. If you remove the `vue.config.js`, the paths are successfully resolved.
We added support for `extractCss` and a few more common options in #52 which we will have another release for soon.

**Comment by alvarosaburido** [comment]
Hi @ZachJW34 thanks for the response, the thing is, `vue.config.js` is often needed to add loaders or extend webpack functionality.  In the specific case of a project I'm working on, it needs to extend assets loader between others so I cannot get rid of it. Is there a way to make them coexist?

**Comment by ZachJW34** [comment]
We plan on adding all configuration options to make our plugin equivalent (or as close) to what vue-cli provides, but for now we don't support the custom `vue.config.js`. The changes we make to the config passed to the Vue CLI are necessary to get Vue working inside an Nx workspace (check them out [here](https://github.com/ZachJW34/nx-plus/blob/0c44221ae6ac4a2d72acd4b22752a57ff74da87c/libs/vue/src/builders/browser/builder.ts#L38)), so as of now the `workspace.json` is the only place for Vue configuration options. We recently merged a PR (#56) to have our builders throw an error if Vue options exist inside the root `package.json` or if a `vue.config.js` exists inside the root of a generated Vue app to help avoid some confusion.
